src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,57 @@
+@use "../utils/var.scss";
+
+.watch {
+  width: $watch-area;
+  height: $watch-area;
+  background-color: $color-watch;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50px;
+  border: 3px solid $color-black;
+
+  &__arrow {
```

Comment:
as I understand it's the center of the stopwatch. In this case, why `arrow`?

Reply:
This is a simple name class before the final result. 
I fixed